content/python/encyclopedia/classes-in-python.md

```diff
@@ -0,0 +1,47 @@
+<!--
+---
```

Comment:
Please add the Title field, [as in the template](https://github.com/Codecademy/codepedia-content/blob/main/documentation/encyclopedia-entry-template.md)